src/style.css

```
@@ -0,0 +1,46 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars__star {
+  list-style: none;
+  display: inline-block;
+}
+
+.stars__container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.stars {
+  position: relative;
+  margin: 0;
+  padding: 0;
+  height: 16px;
+}
+
+.stars--0 .stars__star,
```

delete all classes and use stars__star  instead
```suggestion
.stars__star {
  background-image: url("./images/star.svg");
  ...
```